include/zephyr/bluetooth/mesh/blob_cli.h

```diff
@@ -299,7 +303,7 @@ struct bt_mesh_blob_cli {
 		struct k_work_delayable retry;
 		/* Represents Client Timeout timer in a timestamp. Used in Pull mode only. */
 		int64_t cli_timestamp;
-		struct k_work complete;
+		struct k_work_delayable complete;
```

Comment:
Shouldn't this change go over breaking API change procedure? https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html#introducing-breaking-api-changes

Reply:
I guess this comment is not valid anymore.